### PR TITLE
Updated Jump Statements Lesson in Java

### LINKED ISSUE #5: new lesson: Jumps Statements
### ⭐ Request Type

Add remaining documentation for the lesson `Jumps Statements` in the [file](https://github.com/Utkarsh1504/DSA-Java/blob/main/lessons/jumps.md)

### 💻 Location

This lesson is present under the
Section: `3`
Lesson: `E`
Location Order: `3E`
File-Name: `jumps.md`

### ❌ Additional Info

[here](https://github.com/Utkarsh1504/DSA-Java/blob/main/lessons/operators.md) you can take the help of other markdown files to have an idea in which format you should write.
This lesson must consist of at least the following topic:
1. break + code examples + explanation
2. continue + code examples + explanation
3. break vs continue
4. break vs continue vs return

### PR DESCRIPTION
**Pull request by Ayushi7456**
Fixes #5 
Added Jumps statement Lesson.
 Issue name - new lesson: jumps statements

#### What changes made
- added different jumps statement with code examples and explanation.
- added syntax with examples.
- added comments to make the lesson more understandable.

#### Checklist

- [x] Added description of change
- [x] Added code examples, test must pass
- [x] Added documentation so that the program is self-explanatory and educational.
- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

**Comment by Utkarsh1504** [comment]
@Ayushi7456 I think you should remove the whole **Conditional Jumps** section from the lesson as it is already covered in this lesson https://utkarsh1504.github.io/DSA-Java/io-conditionals

Just covered the below topics in your Pull request:

- break + code examples + explanation
- continue + code examples + explanation
- break vs continue (comparison)
- break vs continue vs return

you if face any issue during the contribuion you ask in the discussion forum.

Thank you for your hard work.🎈

File: lessons/jumps.md [hunk 1]
```diff
@@ -7,22 +7,284 @@ description: "Learn Java"
...
+       case 3:   
+         System.out.println("i is three.");
+         break;
+       default:
+         System.out.println("i is greater than 2.");
+     }
+   }
+}
+```
+Output:  `i is greater than 2.`
```
Comment (Utkarsh1504) [inline lessons/jumps.md:232]: please remove this section.

**Review by Utkarsh1504 — CHANGES_REQUESTED** [review]
all the things look good to me, just do the requested changes and your PR is ready for merge

**Comment by Ayushi7456** [comment]
I have made the changes. Please review and let me know if it's fine now.

File: lessons/jumps.md [hunk 2]
```diff
@@ -6,23 +6,167 @@ section: "Learn Java"
...
+// Initially loop is set to run from 1-5
+for(int i=1; i<=5; i++){
+// terminate loop when i is 4.
+      if(i=4){
+        break;
+        System.out.print(i);
+        }
+    }
+  }
+}
```
Comment (Utkarsh1504) [inline lessons/jumps.md:36]:
```suggestion
public class HelloWorld {
    public static void main(String[] args) {
        // Initially loop is set to run from 1-5
        for(int i=1; i<=5; i++){
            // terminate loop when i is 4.
            if(i==4){
            break;
            System.out.print(i);
            }
        }
    }
}
```
please fix the indentation, a small tweak in the code.

File: lessons/jumps.md [hunk 3]
```diff
@@ -6,23 +6,167 @@ section: "Learn Java"
...
+ - `break`
+ - `continue` 
+ - `return`
+ 
+ These three statements transfer control to other part of the program.
+ 
+Let's see one by one how it works.
 
-### **Unconditional Jumps**
+## **break**
```
Comment (Utkarsh1504) [inline lessons/jumps.md:20]:
```suggestion
## **Break Statements**
```

File: lessons/jumps.md [hunk 4]
```diff
@@ -6,23 +6,167 @@ section: "Learn Java"
...
+Output:    `1 2 3`
 
-### **Conditional Jumps**
+In Java, `break` is majorly used for:
+- To exit a loop.
+- Terminate a sequence in a `switch` statement.
 
-Conditional jumps are used to jump to a specific line in a program.
-It has two parameters, the line number to jump to and a condition. there are two types of conditional jumps:
+## **continue**
```
Comment (Utkarsh1504) [inline lessons/jumps.md:45]:
```suggestion
## **Continue Statement**
```

File: lessons/jumps.md [hunk 5]
```diff
@@ -6,23 +6,167 @@ section: "Learn Java"
...
+// Initially loop is set to run from 1-5
+for(int i=1; i<=6; i++){
+// terminate loop when i is 4.
+      if(i=4){
+        continue;
+        System.out.print(i);
+        }
+    }
+  }
+}
```
Comment (Utkarsh1504) [inline lessons/jumps.md:61]:
```suggestion
public class HelloWorld {
    public static void main(String[] args) {
        // Initially loop is set to run from 1-5
        for(int i=1; i<=6; i++){
            // terminate loop when i is 4.
            if(i==4){
            continue;
            System.out.print(i);
            }
        }
    }
}
```

File: lessons/jumps.md [hunk 6]
```diff
@@ -6,23 +6,167 @@ section: "Learn Java"
...
+   // Method 1
+	public static int sumFunction(int a, int b) {
+		int sum = a + b;
+	// Since return type of sunFunction method is integer so this method should return integer value	
+		return sum;
+	}
+	
+	// Main driver method
+	public static void main(String[] args)
+	{
```
Comment (Utkarsh1504) [inline lessons/jumps.md:100]:
```suggestion
      public static void main(String[] args) {
```

File: lessons/jumps.md [hunk 7]
```diff
@@ -6,23 +6,167 @@ section: "Learn Java"
...
+It is used to **exit** from a method, with or without a value. Usage of **return keyword** as there exist two ways as listed below as follows:
+
+-   **Case 1:**  Methods returning a value
+-   **Case 2:**  Methods not returning a value
+
+
+##  **Methods returning a value**
+```java
+// Main method
+class  CodeExample{
```
Comment (Utkarsh1504) [inline lessons/jumps.md:90]:
```suggestion
class CodeExample {
```

File: lessons/jumps.md [hunk 8]
```diff
@@ -6,23 +6,167 @@ section: "Learn Java"
...
+| break             | continue                                                                |
+| ----------------- | --------------- |
+|The break statement is used to terminate the loop immediately. | The continue statement is used to skip the current iteration of the loop.|
+| break keyword is used to indicate break statements in java programming. | continue keyword is used to indicate continue statement in java programming.|
+| We can use a break with the switch statement. | The continue statement brings the next iteration early. |
+| It stops the execution of the loop. | It does not stop the execution of the loop.|
+
+
+
+## **return**
```
Comment (Utkarsh1504) [inline lessons/jumps.md:79]:
```suggestion
## **Return Statement**
```

**Review by Utkarsh1504 — CHANGES_REQUESTED** [review]
@Ayushi7456 Please fix the indentation and some suggested changes, and PR is ready to merge
Thanks

**Comment by Utkarsh1504** [comment]
@Ayushi7456 Thanks a lot, really appreciate your work.🎈🎉

**Review by Utkarsh1504 — APPROVED** [review]

**Pull request merged**